integration-libs/opf/cta/components/opf-cta-scripts/opf-cta-scripts.service.ts

```diff
@@ -185,8 +180,8 @@ export class OpfCtaScriptsService {
         CtaScriptsLocation.ORDER_HISTORY_PAYMENT_GUIDE,
       [CmsPageLocation.ORDER_CONFIRMATION_PAGE]:
         CtaScriptsLocation.ORDER_CONFIRMATION_PAYMENT_GUIDE,
-      [CmsPageLocation.PDP_PAGE]: CtaScriptsLocation.PDP_QUICK_BUY,
-      [CmsPageLocation.CART_PAGE]: CtaScriptsLocation.CART_QUICK_BUY,
+      [CmsPageLocation.PDP_PAGE]: CtaScriptsLocation.PDP_MESSAGING,
+      [CmsPageLocation.CART_PAGE]: CtaScriptsLocation.CART_MESSAGING,
     };
     return this.cmsService.getCurrentPage().pipe(
```

Comment:
Detecting the current page will be replace by a diff mechanism using URL developed by Mateusz.
Will wait QB to be merged before making the change.